src/common/Html/index.tsx

```diff
@@ -30,7 +30,12 @@ const Html = ({ store, bundlePath, children }: Props) => {
         <link rel="stylesheet" href="/assets/normalize.css" />
         <link rel="stylesheet" href="/assets/style.css" />
 
+        <meta property="og:image:width" content="2400" />
+        <meta property="og:image:height" content="1260" />
+        <meta property="og:image" content="https://stage.kodim.cz/assets/og-image.png" />
```

Comment:
@podlomar, nenapadá tě prosím, kde dynamicky vzít origin (`https://stage.kodim.cz`), aby nemusel být zahardcodovaný? Napadlo mě, jestli to není `serverUrl` v `server-config.json5`, ale nevím, jak máš nastavený server, tak se možná pletu.

Reply:
Jj, mělo by stačit to serverUrl

Reply:
Vyřešeno zde 5a4ea0f420da84c46fcd1b25659829ad1713a065 a tím pádem za mě ready na merge.